script/build.js

```diff
@@ -31,113 +30,51 @@ const {
-options.destination = path.resolve(__dirname, `../build/${options.buildtype}`);
-
-if (isHerokuBuild) {
-  const applyHerokuOptions = require('./heroku-helper');
-  applyHerokuOptions(options);
-}
-
-switch (options.buildtype) {
-  case 'development':
-  // No extra checks needed in dev.
-    break;
-
-  case 'staging':
-    break;
-
-  case 'production':
-    if (options['no-sanity-check-node-env'] === false) {
-      if (env !== 'prod') {
-        throw new Error(`buildtype ${options.buildtype} expects NODE_ENV to be production, not '${process.env.NODE_ENV}'`);
-      }
-    }
-    break;
-
-  case 'devpreview':
-  case 'preview':
-    options['brand-consolidation-enabled'] = true;
-    break;
-
-  default:
-    throw new Error(`Unknown buildtype: '${options.buildtype}'`);
-}
 const manifests = getAppManifests(path.join(__dirname, '..'));
 
 let manifestsToBuild = manifests;
-if (options.entry) {
-  const entryNames = options.entry.split(',').map(name => name.trim());
+if (BUILD_OPTIONS.entry) {
+  const entryNames = BUILD_OPTIONS.entry.split(',').map(name => name.trim());
   manifestsToBuild = manifests
     .filter(manifest => entryNames.includes(manifest.entryName));
 }
 
 const webpackConfig = webpackConfigGenerator(
-  options,
+  BUILD_OPTIONS,
   getWebpackEntryPoints(manifestsToBuild)
 );
 
 // Custom liquid filter(s)
 liquid.filters.humanizeDate = (dt) => moment(dt).format('MMMM D, YYYY');
 
-
 // Set up Metalsmith. BE CAREFUL if you change the order of the plugins. Read the comments and
 // add comments about any implicit dependencies you are introducing!!!
 //
-smith.source(sourceDir);
-smith.destination(options.destination);
+smith.source(`${BUILD_OPTIONS.contentRoot}/pages`);
+smith.destination(BUILD_OPTIONS.destination);
 
 // This lets us access the {{buildtype}} variable within liquid templates.
 smith.metadata({
-  buildtype: options.buildtype,
-  // @deprecated - We use a separate Metalsmith directory for VA.gov. We shouldn't ever need this info in Metalsmith files.
-  mergedbuild: !!options['brand-consolidation-enabled']
+  buildtype: BUILD_OPTIONS.buildtype,
+  mergedbuild: !!BUILD_OPTIONS['brand-consolidation-enabled'] // @deprecated - We use a separate Metalsmith directory for VA.gov. We shouldn't ever need this info in Metalsmith files.
 });
 
 // To block an app from production add the following to the below list:
 //  ignoreList.push('<path-to-content-file>');
 const ignore = require('metalsmith-ignore');
 
 const ignoreList = [];
-if (options.buildtype === 'production') {
+if (BUILD_OPTIONS.buildtype === 'production') {
   manifests.filter(m => !m.production).forEach(m => {
     ignoreList.push(m.contentPage);
   });
   ignoreList.push('veteran-id-card/how-to-get.md');
   ignoreList.push('veteran-id-card/how-to-upload-photo.md');
+  ignoreList.push('education/complaint-tool.md');
+  ignoreList.push('disability-benefits/apply/form-526-disability-claim.md');
```

Comment:
also curious for an update here if it's relevant. I was under (a potentially outdated) impression that this was meant to go live behind a password, just hidden from navigation.

Reply:
Ah thank! Removing both of these from the ignore-list. Normally that list hardly ever changes, so I'm surprised there was any difference between them.